src/components/FileTree.tsx

```diff
@@ -26,10 +26,11 @@ function FileTreeNode({ file, onClick, rightAction, selectedFiles }: FileTreeBas
   if (!file) {
     return null;
   }
-  const isFolder = !!file.children;
+
+  const { isFolder } = file;
 
   // Hide DOT_FILES
-  if (file.name?.startsWith('.')) {
+  if (file.name.startsWith('.')) {
```

Comment:
Maybe we can also have a flag for this text

`isDotfile: boolean`  // note that this should exist in both file and directory.